rest/route/subscription_test.go

```diff
@@ -190,6 +191,14 @@ func (s *SubscriptionRouteSuite) TestProjectSubscription() {
 	s.Require().Len(subs.Result, 1)
 	sub := subs.Result[0].(*model.APISubscription)
 	s.Equal("new type", model.FromAPIString(sub.ResourceType))
+
+	// delete the subscription
+	d := &subscriptionDeleteHandler{id: id}
+	_, err = d.Execute(ctx, s.sc)
```

Comment:
The reason it's not checked is because the execute method never returns anything other than an empty struct. The status codes are handled by the middleware